packages/twenty-server/test/utils/create-app.ts

```diff
@@ -31,7 +31,7 @@ export const createApp = async (
     imports: [AppModule],
   });
 
-  if (!!config.moduleBuilderHook) {
+  if (config.moduleBuilderHook) {
```

Comment:
@thaisguigon Hmm careful with that, in this case it's not causing any issue, but when you're doing that in a react-native three it will cause error, it should not happen on react but can too:

This will cause issue (empty string is not allowed in the three):
```react
<>
  {myString && <></>}
</>
```

But this is not causing issue:
```react
<>
  {!!myString && <></>}
</>
```

Reply:
This custom rule (`explicit-boolean-predicates-in-if`) only applies to `if` statements, it won't flag conditions in JSX.